tasks/actions.yaml

```diff
@@ -0,0 +1,176 @@
+variables:
+  - name: REGISTRY1_USERNAME
+    default: ""
+  - name: REGISTRY1_PASSWORD
+    default: ""
+  - name: GH_TOKEN
+    default: ""
+  - name: CHAINGUARD_IDENTITY
+    default: ""
+  - name: CHART_PATH
+    default: chart/
+  - name: GROUP_NAME
+    default: package
+  - name: COMMON_ZARF
+    default: "true"
+  - name: TYPE
+    default: ""
+  - name: FLAVOR
+    default: ""
+  - name: OPTIONS
+    default: ""
+  - name: GO_VERSION
+    default: 1.23.1
+
+tasks:
+  - name: debug-output
+    actions:
+      - description: Print basic debug info for a k8s cluster
+        cmd: |
+          echo "::group::kubectl get all"
+          uds zarf tools kubectl get all -A || echo "failed" | tee /tmp/debug-k-get-all.log
+          echo "::endgroup::"
+          echo "::group::kubectl get pv,pvc"
+          uds zarf tools kubectl get pv,pvc -A || echo "failed" | tee /tmp/debug-k-get-pv-pvc.log
+          echo "::endgroup::"
+          echo "::group::kubectl get package"
+          uds zarf tools kubectl get package -A || echo "failed" | tee /tmp/debug-k-get-package.log
+          echo "::endgroup::"
+          echo "::group::kubectl get events"
+          uds zarf tools kubectl get events -A --sort-by='.lastTimestamp' || echo "failed" | tee /tmp/debug-k-get-events.log
+          echo "::endgroup::"
+          echo "::group::kubectl describe nodes"
+          uds zarf tools kubectl describe nodes k3d-uds-server-0 || echo "failed" | tee /tmp/debug-k-describe-node.log
+          echo "::endgroup::"
+
+  - name: clean-gh-runner
+    actions:
+      - description: Clean up space on the GitHub runner
+        cmd: |
```

Comment:
Given this is potentially destructive could we have this wrapped in an if `CI` env var or similar exists?

Reply:
my personal opinion is you'd have to be going pretty out of your way to run this "accidentally", but i can update it.